src/components/staking/LiValidator.vue

```diff
@@ -77,6 +89,23 @@ export default {
       default: () => "returns"
     }
   },
+  computed: {
+    status() {
+      if (
+        this.validator.jailed ||
+        this.validator.tombstoned ||
+        this.validator.status === 0
+      )
+        return `Inactive`
+      return `Active`
+    },
+    status_detailed() {
```

Comment:
this is duplicated. is it going to come from Hasura?

Reply:
Yes, both status and status_detailed.